apps/prairielearn/src/lib/editors.ts

```diff
@@ -851,12 +851,15 @@ export class CourseInstanceAddEditor extends Editor {
 export class QuestionAddEditor extends Editor {
   public readonly uuid: string;
 
-  constructor(params: BaseEditorOptions) {
+  files?: Record<string, string>;
+
+  constructor(params: BaseEditorOptions & { files?: Record<string, string> }) {
```

Comment:
Tagging this for attention from @nwalters512 as a change to the core editor code.

Reply:
Discussed this afternoon.